Foundation.hs

```diff
@@ -148,11 +148,12 @@ instance YesodAuth App where
     authHttpManager = httpManager
 
 buildUser :: Creds m -> Maybe User
-buildUser (Creds _ csId csExtra) =
+buildUser (Creds csIdent csPlugin csExtra) =
```

Comment:
These are backwards according to http://hackage.haskell.org/package/yesod-auth-1.4.1.3/docs/Yesod-Auth.html#v:Creds